packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -506,17 +526,18 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     styles: ComponentSlotStylesInput,
     rtl: boolean,
     getToggleButtonProps: (options?: GetToggleButtonPropsOptions) => any,
+    getLabelProps: (options?: GetLabelPropsOptions) => any,
   ): JSX.Element {
-    const { triggerButton } = this.props
+    const { label, triggerButton } = this.props
     const content = this.getSelectedItemAsString(this.state.value)
-
+    const labelProps = getLabelProps()
     const triggerButtonProps = getToggleButtonProps({
       onFocus: this.handleTriggerButtonOrListFocus,
       onBlur: this.handleTriggerButtonBlur,
       onKeyDown: e => {
         this.handleTriggerButtonKeyDown(e, rtl)
       },
-      'aria-label': content,
+      'aria-label': undefined,
```

Comment:
no need for label here. it will narrate either the label text or if missing the placeholder/value in the button content.

Reply:
not sure I understand.. are we overriding existing `aria-label` or why are we setting it to undefined?

Reply:
We are overriding the existing one. The existing one comes from Downshift and we don't want that one. We were overriding it before with the content, but having an `aria-label` on a button with content is enough for screen readers to read it.